src/runtime/rocm/rocm_device_api.cc

```diff
@@ -44,7 +44,11 @@ class ROCMDeviceAPI final : public DeviceAPI {
         value = 64;
         break;
       }
-      case kComputeVersion: return;
+      case kComputeVersion:
+        hipDeviceProp_t prop;
+        ROCM_CALL(hipGetDeviceProperties(&prop, ctx.device_id));
+        value = prop.gcnArch;
+        return;
```

Comment:
You need break here, note how *rv get assigned in line 53

Reply:
rookie mistake